market_maker_keeper/leverjfutures_market_maker_keeper_v2.py

```diff
@@ -0,0 +1,331 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2021 mitakash
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import logging
+import sys
+from typing import List
+from math import log10
+from market_maker_keeper.band import Bands, NewOrder
+from market_maker_keeper.control_feed import create_control_feed
+from market_maker_keeper.limit import History
+from market_maker_keeper.order_book import OrderBookManager
+from market_maker_keeper.order_history_reporter import create_order_history_reporter
+from market_maker_keeper.price_feed import PriceFeedFactory
+from market_maker_keeper.reloadable_config import ReloadableConfig
+from market_maker_keeper.spread_feed import create_spread_feed
+from market_maker_keeper.util import setup_logging
+from pymaker.lifecycle import Lifecycle
+from pymaker.numeric import Wad
+from pyexchange.leverjfutures_v2 import LeverjFuturesAPI, Order
+from web3 import Web3, HTTPProvider
+from pymaker.keys import register_keys
+from decimal import *
+
+_context = Context(prec=1000, rounding=ROUND_DOWN)
+
+
+class LeverjMarketMakerKeeper:
```

Comment:
This class can inherit from: https://github.com/makerdao/market-maker-keeper/blob/master/market_maker_keeper/cex_api.py for further simplification

Reply:
Not sure about this, need further discussion with you about how to implement this and if it's a good idea